apps/web/modules/users/views/users-type-public-view.getServerSideProps.tsx

```diff
@@ -223,6 +232,13 @@ async function getUserPageProps(context: GetServerSidePropsContext) {
     } as const;
   }
 
+  // if Org allows SEOIndexable then only take user's SEOIndexable settings.
+  const allowSEOIndexing = org
+    ? user?.profile?.organization?.organizationSettings?.allowSEOIndexing
+      ? user?.allowSEOIndexing
+      : false
+    : user?.allowSEOIndexing;
+
```

Comment:
Nah, I don't think this is correct. Currently, the personal profile is also SEO indexable if the organization profile SEO indexing is enabled. Also pls remove comment